src/test/java/chess/RookTest.java

```diff
@@ -23,9 +26,80 @@ public void testBlackRookToString() {
     }
+    @Test
+    public void testMoveUpRook() {
+        ChessBoard chessBoard1 = new ChessBoard();
+        chessBoard1.crateAnEmptyChessBoard();
+        ChessBoard chessBoard = new ChessBoard(new Rook(Color.WHITE,new Position(4,4),chessBoard1));
+        Rook piece = (Rook) chessBoard.getBoard()[4][4];
+        boolean po= piece.move(new Position(7,4));
+        assertTrue(po);
+    }
+    @Test
+    public void testMoveDownRook() {
+        ChessBoard chessBoard1 = new ChessBoard();
+        chessBoard1.crateAnEmptyChessBoard();
+        ChessBoard chessBoard = new ChessBoard(new Rook(Color.WHITE,new Position(4,4),chessBoard1));
+        Rook piece = (Rook) chessBoard.getBoard()[4][4];
+        boolean po= piece.move(new Position(0,4));
+        assertTrue(po);
+    }
+    @Test
+    public void testMoveLeftRook() {
+        ChessBoard chessBoard1 = new ChessBoard();
+        chessBoard1.crateAnEmptyChessBoard();
+        ChessBoard chessBoard = new ChessBoard(new Rook(Color.WHITE,new Position(4,4),chessBoard1));
+        Rook piece = (Rook) chessBoard.getBoard()[4][4];
+        boolean po= piece.move(new Position(4,0));
+        assertTrue(po);
+    }
+    @Test
+    public void testMoveRightRook() {
+        ChessBoard chessBoard1 = new ChessBoard();
+        chessBoard1.crateAnEmptyChessBoard();
+        ChessBoard chessBoard = new ChessBoard(new Rook(Color.WHITE,new Position(4,4),chessBoard1));
+        Rook piece = (Rook) chessBoard.getBoard()[4][4];
+        boolean po= piece.move(new Position(4,7));
+        assertTrue(po);
+    }
+
+    @Test
+    public void testMoveDownRookBehindEnemy() {
+        ChessBoard chessBoard1 = new ChessBoard();
+        chessBoard1.crateAnEmptyChessBoard();
+        Rook proof =new Rook(Color.WHITE,new Position(4,4),chessBoard1);
+        chessBoard1.addPiece(proof);
+        chessBoard1.addPiece(new Queen(Color.BLACK,new Position(1,4)));
+        Rook piece = (Rook) chessBoard1.getBoard()[4][4];
+        boolean po= piece.move(new Position(0,4));
+        assertFalse(po);
+    }
+    @Test
+    public void testMoveDownRookAllied() {
+        ChessBoard chessBoard1 = new ChessBoard();
+        chessBoard1.crateAnEmptyChessBoard();
+        Rook proof =new Rook(Color.WHITE,new Position(4,4),chessBoard1);
+        chessBoard1.addPiece(proof);
+        chessBoard1.addPiece(new Queen(Color.WHITE,new Position(1,4)));
+        Rook piece = (Rook) chessBoard1.getBoard()[4][4];
+        boolean po= piece.move(new Position(1,4));
+        assertFalse(po);
+    }
+    @Test
+    public void testMoveUpRookBehindAllied() {
+        ChessBoard chessBoard1 = new ChessBoard();
+        chessBoard1.crateAnEmptyChessBoard();
+        Rook proof =new Rook(Color.WHITE,new Position(4,4),chessBoard1);
+        chessBoard1.addPiece(proof);
+        chessBoard1.addPiece(new Queen(Color.WHITE,new Position(6,4)));
+        Rook piece = (Rook) chessBoard1.getBoard()[4][4];
+        boolean po= piece.move(new Position(7,4));
```

Comment:
don't forget to add spaces after coma

Reply:
i made the changes could you review it please?